flair/models/biomedical_entity_linking.py

```diff
@@ -21,17 +21,29 @@
 from tqdm import tqdm
 
 import flair
-from flair.data import EntityLinkingLabel, Label, Sentence, Span
+from flair.data import EntityLinkingCandidate, EntityLinkingLabel, Label, Sentence, Span
 from flair.datasets import (
-    CTD_CHEMICAL_DICTIONARY,
-    CTD_DISEASE_DICTIONARY,
+    CTD_CHEMICALS_DICTIONARY,
+    CTD_DISEASES_DICTIONARY,
     NCBI_GENE_HUMAN_DICTIONARY,
     NCBI_TAXONOMY_DICTIONARY,
 )
-from flair.datasets.biomedical import ParsedBiomedicalEntityLinkingDictionary
+from flair.datasets.biomedical import (
+    AbstractBiomedicalEntityLinkingDictionary,
+    ParsedBiomedicalEntityLinkingDictionary,
+)
 from flair.embeddings import TransformerDocumentEmbeddings
 from flair.file_utils import cached_path
 
+FAISS_VERSION = "1.7.4"
+
+try:
+    import faiss
+except ImportError as error:
+    raise ImportError(
+        f"You need to install to run the biomedical entity linking: `pip faiss faiss-cpu=={FAISS_VERSION}`"
```

Comment:
Typo: Install command should be "pip install faiss-cpu.." (instead of "pip faiss faiss-cpu").
 
Moreover, I would recommend to adjust the warning and refer to the GPU version of faiss too, i.e. add "pip install faiss-gpu..."

Reply:
I have removed the option to place the index on the GPU.
Large dictionaries require a lot of GPU RAM and unless we offer some compression it does not make too much sense.
We can leave it to as a next feature in a separate PR.